Removed some of the printing during runtime

psavery opened the pull request:
Removed dot printing, space printing, and '<' or '>'
printing during runtime. It's somewhat inconvenient
to scroll all the way back up to the top to see
messages at the beginning of the run. So this fixes
that issue.

greglandrum commented:
This status printing was there for cases where the calculations took a while to run (so that you could see what was going on) and I think they could still be useful.
How about adding a verbose option (defaulting to false) that controls this output?

psavery commented:
Yes I can do that!

psavery committed:
Removed some of the printing during runtime

All the printing can be brought back if the keyword "show progress"
is added to the input file.

psavery commented:
Okay. This has been updated. Now, there is another global variable called "print_progress" that is a boolean. If the user adds "show progress" to anywhere in the input file, "print_progress" gets set to true and the progress gets printed just as it did before. By default, it does not print the progress. (a.k.a. print_progress is set to false by default)
